pkg/app/run_test.go

```diff
@@ -259,7 +259,11 @@ SUBCOMMANDS
   service list [<flags>]
     List Fastly services
 
-    -j, --json  Render output as JSON
+        --direction=ascend   Direction in which to sort results
+    -j, --json               Render output as JSON
+        --page=PAGE          The start page
```

Comment:
"Start" doesn't work so well for me because the idea of pagination is that the selection of results you get is a 'page', right?  'start' page implies you get more than one.  Here it's the case that if you set page=2 and per-page=100, you get results 101-200, correct?   So I'd describe that as "Page number of data set to fetch" or something like that